frontend/src/components/episode/episode.tsx

```diff
@@ -48,6 +49,7 @@ const Episode: React.FC = () => {
   useEffect(() => {
     dispatch(episodeActions.loadCommentsByEpisodeId(Number(id)));
     dispatch(episodeActions.loadEpisode(Number(id)));
+    dispatch(episodeActions.loadPodcast(Number(id)));
```

Comment:
```suggestion
    dispatch(episodeActions.loadEpisodePayload(Number(id)));
```
we should use only one action